internal/facilities/retrieve.go

```diff
@@ -37,23 +24,56 @@ func (c *Client) Retrieve() *cobra.Command {
   metal facilities get`,
 
 		RunE: func(cmd *cobra.Command, args []string) error {
+			var (
+				facilityList *metalv1.FacilityList
+				err          error
+			)
 			cmd.SilenceUsage = true
-			facilities, _, err := c.Service.List(c.Servicer.ListOptions(nil, nil))
-			if err != nil {
-				return fmt.Errorf("Could not list Facilities: %w", err)
+			includes := []string{}
+			excludes := []string{}
+			if projectID != "" {
+				facilityList, _, err = c.Service.FindFacilitiesByProject(context.Background(), projectID).Include(c.Servicer.Includes(includes)).Exclude(c.Servicer.Excludes(excludes)).Execute()
+				if err != nil {
+					return fmt.Errorf("could not list Project Metal Facilities: %w", err)
+				}
+			} else if organizationID != "" {
+				facilityList, _, err = c.Service.FindFacilitiesByOrganization(context.Background(), organizationID).Include(c.Servicer.Includes(includes)).Exclude(c.Servicer.Excludes(excludes)).Execute()
+				if err != nil {
+					return fmt.Errorf("could not list Organization Metal Facilities: %w", err)
+				}
+
+			} else {
+				inc := []metalv1.FindFacilitiesIncludeParameterInner{}
+				exc := []metalv1.FindFacilitiesIncludeParameterInner{}
+				facilityList, _, err = c.Service.FindFacilities(context.Background()).Include(inc).Exclude(exc).Execute()
+				if err != nil {
+					return fmt.Errorf("could not list All Metal Facilities: %w", err)
+				}
 			}
+			facilities := facilityList.GetFacilities()
 			data := make([][]string, len(facilities))
 
 			for i, facility := range facilities {
 				var metro string
 				if facility.Metro != nil {
-					metro = facility.Metro.Code
+					metro = facility.Metro.GetCode()
 				}
-				data[i] = []string{facility.Name, facility.Code, metro, strings.Join(facility.Features, ",")}
+
+				facilityFeatures := facility.GetFeatures()
+				var stringFeatures []string
+				for _, feature := range facilityFeatures {
+					stringFeatures = append(stringFeatures, string(feature))
+				}
+				data[i] = []string{facility.GetName(), facility.GetCode(), metro, strings.Join(([]string)(stringFeatures), ",")}
 			}
 			header := []string{"Name", "Code", "Metro", "Features"}
-
 			return c.Out.Output(facilities, header, &data)
 		},
 	}
+
+	retrieveFacilitesCmd.Flags().StringVarP(&projectID, "project-id", "p", "", "The project's UUID. This flag is required, unless specified in the config created by metal init or set as METAL_PROJECT_ID environment variable.")
+	retrieveFacilitesCmd.Flags().StringVarP(&organizationID, "organization-id", "O", "", "UUID of the organization")
```

Comment:
It looks like the packngo implementation was using the equivalent of [this endpoint](https://github.com/equinix/equinix-sdk-go/blob/main/services/metalv1/api_facilities.go#L59) and did not support CLI flags.  We should replicate what the packngo implementation did instead of adding new features & changing default behavior.